runtime/compiler/env/J9CPU.hpp

```diff
@@ -45,12 +45,17 @@ class OMR_EXTENSIBLE CPU : public OMR::CPUConnector
    CPU() : OMR::CPUConnector() {}
    CPU(const OMRProcessorDesc& processorDescription) : OMR::CPUConnector(processorDescription) {}
 
+   static OMRProcessorDesc _featureMasks;
+   static bool _shouldUseFeatureMasks; 
+   static void applyFeatureMasks(OMRProcessorDesc& processorDescription, const uint32_t* enabledFeatures, size_t size); 
```

Comment:
I would like some comments on the meaning of the parameters and what this function is supposed to do